lib/ovh.es6.js

```diff
@@ -1,6 +1,6 @@
 /**
- * Copyright (c) 2014 OVH SAS
- * Copyright (c) 2012 - 2013 Vincent Giersch
```

Comment:
Alright I'll take care of that. But I think it was important to note who is maintaining this code but ... if you think that's necessary it's the same for me :)